crates/pera-framework/packages/pera-system/sources/dwallet_2pc_mpc_ecdsa_k1.move

```diff
@@ -417,6 +453,28 @@ module pera_system::dwallet_2pc_mpc_ecdsa_k1 {
         );
     }
 
+    #[test_only]
```

Comment:
t is more appropriate to place the test logic at the bottom of the file.

Reply:
We don't have this convention in this file for now, and changing it will introduce many changes that are not related to this pr